pkg/cli/common.go

```diff
@@ -13,263 +13,183 @@ import (
-		},
-		cli.StringSliceFlag{
-			Name:  "file, f",
-			Usage: "`pathname or URL` of a Dockerfile",
-		},
-		cli.StringFlag{
-			Name:  "format",
-			Usage: "`format` of the built image's manifest and metadata. Use BUILDAH_FORMAT environment variable to override.",
-			Value: DefaultFormat(),
-		},
-		cli.StringFlag{
-			Name:  "iidfile",
-			Usage: "`file` to write the image ID to",
-		},
-		cli.StringSliceFlag{
-			Name:  "label",
-			Usage: "Set metadata for an image (default [])",
-		},
-		cli.BoolFlag{
-			Name:  "no-cache",
-			Usage: "Do not use existing cached images for the container build. Build from the start with a new set of cached layers.",
-		},
-		cli.StringFlag{
-			Name:  "logfile",
-			Usage: "log to `file` instead of stdout/stderr",
-		},
-		cli.IntFlag{
-			Name:  "loglevel",
-			Usage: "adjust logging level (range from -2 to 3)",
-		},
-		cli.StringFlag{
-			Name:  "platform",
-			Usage: "CLI compatibility: no action or effect",
-		},
-		cli.BoolTFlag{
-			Name:  "pull",
-			Usage: "pull the image if not present",
-		},
-		cli.BoolFlag{
-			Name:  "pull-always",
-			Usage: "pull the image, even if a version is present",
-		},
-		cli.BoolFlag{
-			Name:  "quiet, q",
-			Usage: "refrain from announcing build instructions and image read/write progress",
-		},
-		cli.BoolTFlag{
-			Name:  "rm",
-			Usage: "Remove intermediate containers after a successful build (default true)",
-		},
-		cli.StringFlag{
-			Name:  "runtime",
-			Usage: "`path` to an alternate runtime. Use BUILDAH_RUNTIME environment variable to override.",
-			Value: util.Runtime(),
-		},
-		cli.StringSliceFlag{
-			Name:  "runtime-flag",
-			Usage: "add global flags for the container runtime",
-		},
-		cli.StringFlag{
-			Name:  "signature-policy",
-			Usage: "`pathname` of signature policy file (not usually used)",
-		},
-		cli.BoolFlag{
-			Name:  "squash",
-			Usage: "Squash newly built layers into a single new layer. The build process does not currently support caching so this is a NOOP.",
-		},
-		cli.StringSliceFlag{
-			Name:  "tag, t",
-			Usage: "tagged `name` to apply to the built image",
-		},
-		cli.BoolTFlag{
-			Name:  "tls-verify",
-			Usage: "require HTTPS and verify certificates when accessing the registry",
-		},
-	}
+// BudResults represents the results for Bud flags
+type BudResults struct {
+	Annotation          []string
+	Authfile            string
+	BuildArg            []string
+	CacheFrom           string
+	CertDir             string
+	Compress            bool
+	Creds               string
+	DisableCompression  bool
+	DisableContentTrust bool
+	File                []string
+	Format              string
+	Iidfile             string
+	NoCache             bool
+	Label               []string
+	Logfile             string
+	Loglevel            int
+	Platform            string
+	Pull                bool
+	PullAlways          bool
+	Quiet               bool
+	Rm                  bool
+	Runtime             string
+	RuntimeOpts         []string
```

Comment:
Could you change this to RuntimeFlags to be consistent with the actual option please?